sites/svelte-5-preview/src/routes/docs/content/01-api/05-imports.md

```diff
@@ -115,6 +115,39 @@ Svelte provides reactive `Map`, `Set`, `Date` and `URL` classes. These can be im
 <input bind:value={url.href} />
 ```
 
+## `svelte/events`
+
+Svelte provides a way of imperatively attaching DOM event listeners to elements using the `attach` export from `svelte/events`. This can be used in place of
+imperatively doing `element.addEventListener`, with that benefit that `attach` will allow Svelte to co-ordinate the event through its own event delegation system.
+
+```js
+// @filename: index.ts
+const element: Element = null as any;
+// ---cut---
+import { attach } from 'svelte/events';
+
+attach(element, 'click', () => {
+	console.log('click the element!');
```

Comment:
This sounds like the text for a button, not the text for the reaction of having clicked the button. Suggestion:
```suggestion
	console.log('element was clicked');
```